include/deal.II/grid/grid_tools.h

```diff
@@ -2006,15 +2006,22 @@ namespace GridTools
 
   /**
    * Generates a partitioning of the active cells making up the entire domain
-   * using the same partitioning scheme as in the p4est library. After calling
+   * using the same partitioning scheme as in the p4est library if the flag
+   * blocked is set to true (default behavior of this function). After calling
    * this function, the subdomain ids of all active cells will have values
    * between zero and @p n_partitions-1. You can access the subdomain id of a
    * cell by using <tt>cell-@>subdomain_id()</tt>.
+   *
+   * @note If the flag blocked is set to false, some assumptions of the p4est
```

Comment:
The only change is placing all terminal children of a single parent on a single process, right? Can you change the text here?